Projects/Features/Scene/NewsCardScene/NewsList/NewsCard/NewsCardCore.swift

```diff
@@ -35,6 +36,12 @@ public struct NewsCardEnvironment {
 public let newsCardReducer = Reducer.combine([
   Reducer<NewsCardState, NewsCardAction, NewsCardEnvironment> { state, action, env in
     switch action {
+    case .rightButtonTapped:
+      return Effect(value: ._navigateWebView(state.news.newsLink))
+      
+    case .cardTapped:
+      return Effect(value: ._navigateWebView(state.news.newsLink))
```

Comment:
같은 이펙트를 리턴하는 액션이니 묶는 것도 좋아보입니당
```suggestion
    case .rightButtonTapped, .cardTapped:
      return Effect(value: ._navigateWebView(state.news.newsLink))
```